src/main/java/net/pms/io/OutputParams.java

```diff
@@ -28,103 +27,321 @@
+		return minBufferSize;
+	}
+
+	public void setMinBufferSize(double minBufferSize) {
+		this.minBufferSize = minBufferSize;
+	}
+
+	public double getMinFileSize() {
+		return minFileSize;
+	}
+
+	public void setMinFileSize(double minFileSize) {
+		this.minFileSize = minFileSize;
+	}
+
+	public double getMaxBufferSize() {
+		return maxBufferSize;
+	}
+
+	public void setMaxBufferSize(double maxBufferSize) {
+		this.maxBufferSize = maxBufferSize;
+	}
+
+	public double getTimeseek() {
+		return timeseek;
+	}
+
+	public void setTimeseek(double timeseek) {
+		this.timeseek = timeseek;
+	}
+
+	public double getTimeend() {
+		return timeend;
+	}
+
+	public void setTimeend(double timeend) {
+		this.timeend = timeend;
+	}
+
+	public int getFromFrame() {
+		return fromFrame;
+	}
+
+	public void setFromFrame(int fromFrame) {
+		this.fromFrame = fromFrame;
+	}
+
+	public int getWaitbeforestart() {
+		return waitbeforestart;
+	}
+
+	public void setWaitbeforestart(int waitbeforestart) {
+		this.waitbeforestart = waitbeforestart;
+	}
+
+	public int getToFrame() {
+		return toFrame;
+	}
+
+	public void setToFrame(int toFrame) {
+		this.toFrame = toFrame;
+	}
+
+	public PipeProcess[] getOutput_pipes() {
+		return output_pipes;
+	}
+
+	public void setOutput_pipes(PipeProcess[] output_pipes) {
+		this.output_pipes = output_pipes;
+	}
+
+	public PipeProcess[] getInput_pipes() {
+		return input_pipes;
+	}
+
+	public void setInput_pipes(PipeProcess[] input_pipes) {
+		this.input_pipes = input_pipes;
+	}
+
+	public DLNAMediaAudio getAid() {
+		return aid;
+	}
+
+	public void setAid(DLNAMediaAudio aid) {
+		this.aid = aid;
+	}
+
+	public DLNAMediaSubtitle getSid() {
+		return sid;
+	}
+
+	public void setSid(DLNAMediaSubtitle sid) {
+		this.sid = sid;
+	}
+
+	public int getOutputByteArrayStreamBufferSize() {
+		return outputByteArrayStreamBufferSize;
+	}
+
+	public void setOutputByteArrayStreamBufferSize(int outputByteArrayStreamBufferSize) {
+		this.outputByteArrayStreamBufferSize = outputByteArrayStreamBufferSize;
+	}
+
+	public int getSecondread_minsize() {
+		return secondread_minsize;
+	}
+
+	public void setSecondread_minsize(int secondread_minsize) {
+		this.secondread_minsize = secondread_minsize;
+	}
+
+	public boolean isLog() {
+		return log;
+	}
+
+	public void setLog(boolean log) {
+		this.log = log;
+	}
+
+	public boolean isNoexitcheck() {
+		return noexitcheck;
+	}
+
+	public void setNoexitcheck(boolean noexitcheck) {
+		this.noexitcheck = noexitcheck;
+	}
+
+	public boolean isLosslessaudio() {
+		return losslessaudio;
+	}
+
+	public void setLosslessaudio(boolean losslessaudio) {
+		this.losslessaudio = losslessaudio;
+	}
+
+	public boolean isLossyaudio() {
+		return lossyaudio;
+	}
+
+	public void setLossyaudio(boolean lossyaudio) {
+		this.lossyaudio = lossyaudio;
+	}
+
+	public boolean isNo_videoencode() {
+		return no_videoencode;
+	}
+
+	public void setNo_videoencode(boolean no_videoencode) {
+		this.no_videoencode = no_videoencode;
```

Comment:
This could be a good opportunity to also rename these private variables to our usual convention like `isNoVideoEncode` or just `noVideoEncode`. We already have that convention in this file (like `forceFps and forceType` just below here) so it would be good to be consistent. Let me know if you want me to make that change, or if you disagree

Reply:
@SubJunk done in 49ab5c1